### DISCUSSION TITLE
Future Plans for the Nepetel (Under Construction)

### DISCUSSION THREAD

**Discussion by AvianGeneticist** [endless-sky/endless-sky#8703] [Ideas]
So recently I opened up a VERY early draft PR for the Nepetel (#8702) for things like balance feedback and to drum up general interest in the idea as I slowly plug away at it (I'm used to plugin dev which is far more loosely paced, so I apologise if that's a bit out-of-protocol around here).

However, it's been brought to my attention that... I don't have a story planned for these guys. Individual interactions I'd like the player to have in their space, sure, like a Kestrel-style mission where the player is able to choose how they augment one of their new weapons, as well as a basic idea for a first contact, but besides that I really don't have much planned (I'd like to keep them a bit more "open" for the time being so they can interact with some newer races). So, since I'm trying to not be an indomitable faction lead who keeps all the dev stuff to myself, why not ask the community for their input?

# So, let's start with what I have.
> Quoted passages are actual lore,

Whereas unquoted passages are dev notes/thoughts for alteration.

> The Nepetel are an old race; the first anatomically "modern" Nepetel evolved sometime before the Drak's discovery of the jump drive, and from there they developed planetside extremely slowly. So slowly, in fact, that the first successful permanent satellite in space was established in what would be known on earth as around 20,000 BC.

This is quite a large span of time, but keep in mind that anatomical modernity is the very first step in a sentient species becoming sentient; the time between this first emergence and extraplanetary establishment covers their stone and metal ages, as well as industrial progression culminating in spaceflight (including the development of tech like repulsors that are common among young ES species).

One note I'd like to add before we continue is the fact that the Nepetel, being unicellular organisms, use purely chemical memory instead of electrochemical memory like multicellular beings do. This, combined with the fact that they reproduce by mitosis (binary fission), means that at least one "child" of a parent who's split retains all the memories of their former parent (multiple copies of each memory are usually made, so it's not unlikely for both children to have the memories, but know it *is* possible for only one child to inherit certain memories).
Memories can also be shared like plasmids between individuals, so knowledge propagates *exceedingly* quickly. Only one Nepetel truly has to "learn" something in the way we multicellular folks do; from there, one Nepetel can propagate the knowledge to another, those 2 to 4, those 4 to 8, and so on. As such, large-scale ancestral memories (like the broadcasting of the first successful spaceflights) are remembered by not only the generation that experienced them, but (most of) the generations that follow.

> Over the next 3,000 years or so, the Nepetel expanded to every celestial body in their home system, eventually stagnating once they reached the limit. In that time, instead of building generation spaceships or some similar exploratory vehicle, they expanded inward; all of their home system's planets have a multitude of objects around them, believing the answer would come to them when the time was right, just as cheap and effective spaceflight had graced them as they had seemingly reached the limits of their homeworld's potential.

This seems a little unrealistic, considering a species that's gotten to space is likely particularly expansionist; the reasoning feels a little flimsy. However, it's mostly a setup for the next major defining event in their history:

> The Nepetel were visited about 5,000 years after the last object in their solar system (the star aside) had been colonized by and alien race whom the Nepetel now call the "Lifebringers". These Lifebringers taught them much about the wider galaxy, and - as a parting gift of sorts - gave the Nepetel not only Scram Drives, but extensive knowledge of how to produce them. This ancestral memory still lives *very* strongly in modern Nepetel, and the generosity of these space strangers has shaped their present day overly kind, welcoming disposition towards strangers.

The origins of the Lifebringers are intentionally left vague; I'd like them to be one of those "dead and gone" sorts of races. As a fun trivia note, the only faction that currently fills the requirements to be source of these scram drives are the Korath (they're the only faction we've seen to produce both Scram and Jump drives).

> Over the next 500 years or so, the Nepetel greatly exploded over the dozen or so star systems. Their extensive knowledge of extraplanetary infrastructure built up from aeons of being confined to one order of magnitude at a time (first their home planet, then their home solar system) meant that they expanded so fast that they ended up both meeting and interrupting the Quarg in the process of ringworld building. Though the Quarg and the Nepetel do not have a strong relationship, the Quarg's wisdom and pacifism towards the Nepetel only served to further the Nepetel's inclination that the galaxy was a nice place full of kind people interested in peace and commerce.

I'd like to think it's not unreasonable that the Quarg would be able to be interrupted in the early stages of their project; especially if a native species grew at the explosive rate I imagine the Nepetel did. The Nepetel already have many ancestral memories of being constrained for space, so I highly doubt they'd let the Quarg eat up an entire system, and thus the Quarg likely just have a moderately large station in their turf.

> In modern times, the Nepetel have stagnated again, though some have begun probing what is widely known as the "Jump Drive". They haven't made one yet, nor anything close to a successful prototype, but it is known they are trying.

In the same way humans are attempting to make jump drives, so are the Nepetel, representing a sort of ideological shift in their kind from those who allow life to happen to them to those who seek out life themselves. This is probably spurred by the Quarg (semi-unintentionally) revealing to them that Jump Drives are indeed a thing that exists and are possible, though they're a long way off from actually succeeding in making anything successful (not gonna happen in any playable timescale, I'm talking at least a millenium or 6 here).

# Still here? I admire your persistence. Let's discuss the in-game interactions.

As said above in some parts of their backstory, the Nepetel have had a very positive view of their galactic neighbors so far, so they start notably friendly to the player; so much so that the player can buy their outfits without a license; their ships, however, are locked behind a license (more as a game-balance measure than a lore one).

Their first contact is likely going to consist of bringing one Nepetel to human space to learn human language, then bringing them back. How the player learns to do this is up for debate; it could either be a curious diplomat gesturing hopefully to join the player on their travels, or one of the Quarg explaining to you that they need one individual to learn human language so the wider populous can adopt the knowledge.

Being a T1.5 race (intended to be somewhere between the Hai and Remnant), I'd like the Nepetel to be approachable for new players, though I'd like to avoid just copying the Coalition's "bum around doing jobs for a month" method of license unlocking. One of my ideas was to have some of their missions be diplomatic escort missions; the Nepetel are a race of "soft-hearted fools", so they'd be interested in gifting some of their ships and technology to humanity as a form of welcoming gift. Of course, neither species has particularly regular access to jump drives, so it's up to the player to supply one for these missions to help the ships make it to and from. This includes actually making sure the ship gets there, of course, which could be complicated by the placement of the Nepetel (for example, if Automata space happened to be between the two).

Other mission ideas include helping test technology (the Nepetel are a race that values contributions to science; if you let yourself get bumped around by some of their fancy new guns, they'll probably let you buy your own set), aiding in the construction of a new station (escorting a Cilevikar, and directing it to the place where it will be built up, then providing materials, for example).
I think some mining missions would also fit their heavy, industrial theme; mine X tons of Y by Z and get paid in money and favours.

The Nepetel also have the "Outriders", who are sort of like human Marauders on steroids. Outriders heavily modify their ships, but don't stop at just combat; often, an Outrider variant of a vessel will have entirely reworked capabilities compared to its non-outrider variant. These changes are almost always for the better, though some variants have notable drawbacks (some variants might lose bays, gain noticeably increased drag or reduced hardpoints, and so on). I'd like for their stuff to be a separate license, but I'm unsure how to distinguish it from the base license; ideas are appreciated here. Though I did have the idea that they use special "tokens" you can earn via doing missions for them; kind of like a premium currency on top of credits. Maybe they could have special hyper-rushed missions that only pay a normal amount, but gives you a token/cheque/ticket you can redeem later for an outrider ship of your choice (these would have to be offered rarely, of course; not finite, but a very uncommon reward for being dilligent with hanging in their space and checking the job board/spaceports carefully).

That's all I have for now, but I'll be tweaking things as I see suggestions crop up in the comments. 

If you'd like to engross yourself in further reading (because you obviously aren't tired of staring at text on a screen already), I have a doc where most of these ideas were originally written down:
https://docs.google.com/document/d/1z4kv75v1liylnxXB5gRTqOpqkY8oaVvIBFoTc52dNLI/edit?usp=sharing

With that, I'm heading to bed, because it's 20 after 1 for me. Thanks for reading! I look forward to the community feedback!

**Comment by Arachi-Lover** [comment]
>...meant that they expanded so fast that they ended up both meeting and interrupting the Quarg in the process of ringworld building.

>I'd like to think it's not unreasonable that the Quarg would be able to be interrupted in the early stages of their project; especially if a native species grew at the explosive rate I imagine the Nepetel did. The Nepetel already have many ancestral memories of being constrained for space, so I highly doubt they'd let the Quarg eat up an entire system, and thus the Quarg likely just have a moderately large station in their turf.

I'd really rather we don't because as it stands there are 2 cases where that's happened already (Korath, Bunrodea), and I really don't think it's good for the game's lore for us to just allow this trope of the Quarg having their ringworlds' construction stopped because the species just didn't want them there.

**Reply by AvianGeneticist** [reply]
I'm not a very huge fan of the Quarg just eating an entire system (even if it does or doesn't make sense, it's just something I particularly dislike).

I would be willing to compromise and, say, let the Quarg have their ringworld, but have some Nepetel stations/other objects in the system as well (hence, the Nepetel still own the territory, but the Quarg are still present). Would that be better?

**Reply by Arachi-Lover** [reply]
>I'm not a very huge fan of the Quarg just eating an entire system (even if it does or doesn't make sense, it's just something I particularly dislike).

Regardless, it is important to keep their presense in so as to not just downplay them as some chumps that get told off from building their rings by everyone who bothers asking, and necessary given they're meant to be around everywhere.

>I would be willing to compromise and, say, let the Quarg have their ringworld, but have some Nepetel stations/other objects in the system as well (hence, the Nepetel still own the territory, but the Quarg are still present). Would that be better?

Yes. If you wish to not have a full ring yet you can also probably tweak the timeframe or get working on an idea that'd justify why they'd not have finished so far. At any rate, the new aliens could have stations there yes.

**Comment by lumbar527** [comment]
I think the name changing addition is kind of cool. The Nepetel sound like they would be great!
